src/table.ts

```diff
@@ -99,7 +100,11 @@ export class Table<TableDef extends TableDefinition> {
     return newItem;
   }
 
-  private buildQBData(body: Partial<DocumentFromTable<TableDef>>) {
+  private buildQBData(
+    body: Partial<
+      QueryDocumentFromTable<TableDef> | UpsertDocumentFromTable<TableDef>
+    >,
+  ) {
```

Comment:
Shouldnt this only take UpsertDocumentFromTable<TableDef>

Reply:
I dont think so.
The Update one function takes in a UpsertDocumentFromTable and uses the buildQBData function to create the data field in the request.

Reply:
Im confused. Doesn't that just confirm what I said? The three places using buildQBData are:
updateOne
upsertMany
createOne

all of which use UpsertDocumentFromTable

Reply:
Total brain lapse, yes you are right. My bad

Reply:
All g. Id rather have you say something instead of nothing